Formula/a/azion-cli.rb

```diff
@@ -21,6 +21,9 @@ def install
     ldflags = %W[
       -s -w
       -X github.com/aziontech/azion-cli/pkg/cmd/version.BinVersion=#{version}
+      -X github.com/aziontech/azion-cli/pkg/constants.StorageApiURL=https://storage-api.azion.com
+      -X github.com/aziontech/azion-cli/pkg/constants.AuthURL=https://sso.azion.com/api/user/me
+      -X github.com/aziontech/azion-cli/pkg/constants.ApiURL=https://api.azionapi.net
```

Comment:
can we update test for this?

Reply:
@PatrickMenoti are these flags being used?

Reply:
They are being used for api calls, however they need an authentication token to be accessed (which can be configured through cli commands)